sdk/contentsafety/azure-ai-contentsafety/src/samples/README.md

```diff
@@ -0,0 +1,71 @@
+---
+page_type: sample
+languages:
+    - java
+products:
+    - azure
+    - azure-cognitive-services
+    - azure-cognitive-services
```

Comment:
remember to change this line after https://github.com/Azure/azure-sdk-tools/pull/7346 merged.

Reply:
@zhaiyutong FYI you should be good with azure-ai-content-safety here since that is how the product is registered. I dont believe the product needs to match the SDK package name exactly.